From d08502db67c1a725359a9fce7aca3d8a43d024ac Mon Sep 17 00:00:00 2001
From: suvojeet-sengupta <suvojitsengupta21@gmail.com>
Date: Tue, 6 Oct 2026 21:09:47 +0000
Subject: [PATCH] sepolicy: Don't declare the QSPM HAL attributes

AOSPA's QSSI sepolicy already declares them, so the build failed on
duplicate declarations in pub_policy.cil.
---
 sepolicy/common/public/attributes | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/sepolicy/common/public/attributes b/sepolicy/common/public/attributes
index 9a245e3a..e69de29b 100644
--- a/sepolicy/common/public/attributes
+++ b/sepolicy/common/public/attributes
@@ -1,4 +0,0 @@
-# Attributes for QSPM HAL
-attribute vendor_hal_qspmhal;
-attribute vendor_hal_qspmhal_client;
-attribute vendor_hal_qspmhal_server;
-- 
2.43.0

